apps/meteor/ee/server/lib/engagementDashboard/channels.ts

```diff
@@ -43,15 +43,7 @@ export const findAllChannelsWithNumberOfMessages = async ({
 		options,
 	}).toArray();
 
-	const total =
-		(
-			await Rooms.countChannelsWithNumberOfMessagesBetweenDate({
-				start: convertDateToInt(start),
-				end: convertDateToInt(end),
-				startOfLastWeek: convertDateToInt(startOfLastWeek),
-				endOfLastWeek: convertDateToInt(endOfLastWeek),
-			}).toArray()
-		)[0]?.total ?? 0;
+	const total = await Rooms.countDocuments({});
```

Comment:
> why not just add $count to existing aggregation, and take from there?

I could do that with a `$facet` in the existing aggregation, but since this `$limit` may be performed before the result is complete (in the "middle" of it), we'd carry an object like this starting on that point:

```
rooms: [ { room: {}, messages: 5, lastWeekMessages: []}, ...]
totalCount: 25
```
This would make us need to apply **an extra unwind** to get the data inside the (`rooms`) array, I'm not sure it's worth it.